tests/emailControllertest.js

```diff
@@ -0,0 +1,19 @@
+import chai from 'chai';
```

Comment:
#### Critical
Rename this file.
All test file should have a `.test` append to the name
i.e. emailControllertest should be `emailController.test.js`